EdvClient.js

```diff
@@ -572,7 +571,7 @@ export class EdvClient {
       });
       // send request
       const {httpsAgent} = this;
-      await axios.post(url, capabilityToEnable, {headers, httpsAgent});
+      return await axios.post(url, capabilityToEnable, {headers, httpsAgent});
```

Comment:
Is the result used?  If not, would it possible stack traces be better by not returning here?

Reply:
I'm not sure actually. The jsdoc above it mentions a promise, but it was not returning it before. Might be better to not return it? I'll let more knowledge minds make the call on this one. If this is a security issue then we should not return it, but we do need to update the docs.

Reply:
This is in an async function so a promise will always be returned.

Reply:
I guess the promise wouldn't resolve to an object.  But it's close.

Reply:
it resolves to an axios response which is an object. The issue is does it potentially leak anything. I think returning the response here is fine. @dlongley opinions?

Reply:
This is client code so there isn't any security risk that could be mitigated by not returning the result. However, we don't need to make a change like this, so let's just fix the docs and remove the return value.

Reply:
Changed the docs to show {undefined} https://github.com/digitalbazaar/edv-client/pull/26/commits/d082da0eeca8d3ea5138af15cdb48304bf992d3b